src/eio/linol_eio.ml

```diff
@@ -14,7 +14,10 @@ module IO_eio :
   let return x = x
   let failwith = failwith
   let fail = raise
-  let catch f handler = try f () with exn -> handler exn
+
+  let catch f handler = try f () with exn ->
```

Comment:
format?

Reply:
Too lazy to set up a separate opam switch with the older ocamlformat :-)